### ISSUE TITLE
Ghostery should not block first party content

### ISSUE THREAD

**Issue by bluesmoon** [ghostery/ghostery-extension#1652]
It appears the Ghostery is blocking content from `mpulse.soasta.com` while browsing `mpulse.soasta.com`. I cannot see how this would make sense in any context.
<img width="346" alt="image" src="https://github.com/ghostery/ghostery-extension/assets/35098/8db2f66c-e5a2-43bb-93c4-6883c9183370">

**Comment by bluesmoon** [comment]
Yeah, it's definitely blocking content. I get a blank page because the page JavaScript is blocked.

These are the blocked URLs:
* https://mpulse.soasta.com/concerto/ClientScripts/mpulse.js/mpulse.js?version=62854
* https://mpulse.soasta.com/concerto/img/spacer.gif?version=62854

This is used to fetch and render SPA content on mpulse.soasta.com so it shouldn't be blocked.

It's unlikely that the community lists block these because the only people who actually see these scripts are paying customers of mPulse, and it blocks functionality that they have paid for.

**Comment by bluesmoon** [comment]
<img width="1277" alt="image" src="https://github.com/ghostery/ghostery-extension/assets/35098/116e3bc5-0af9-4dab-8aea-feac29d8b7c5">

**Comment by chrmod** [comment]
Have verify the source of the blockage and it is Easylist Privacy in both cases:
* https://mpulse.soasta.com/concerto/img/spacer.gif?version=62854 is blocked by 
 - https://github.com/easylist/easylist/blob/44dddf7368fd7bf626bee5aa69330c6f4e658cb2/easyprivacy/easyprivacy_general.txt#L2008
* https://mpulse.soasta.com/concerto/ClientScripts/mpulse.js/mpulse.js?version=62854 - https://github.com/easylist/easylist/blob/44dddf7368fd7bf626bee5aa69330c6f4e658cb2/easyprivacy/easyprivacy_general.txt#L1453

**Comment by chrmod** [comment]
The only solution that we can apply is to replace those filters with ones that apply as third party only, that is `/spacer.git$3p` and `/mpulse.js$3p`. But this will help with Ghostery only. As long original filters are in the Easylist Privacy, other adblockers like uBlockOrigin, Brave or AdGuard will also cause same breakage. 
You can try to propose changes to the EasyPrivacy as a solution that will work for all adblockers.

**Comment by bluesmoon** [comment]
Thanks for the investigation.  I've opened an issue on EasyPrivacy to get their feedback.

**Comment by bluesmoon** [comment]
@chrmod it looks like EasyList has closed the issue as "Not Planned". That would leave modifying Ghostery as the only option here.

**Comment by chrmod** [comment]
With community lists it's best to offer exact solution with minimal side-effects. They are most often right to have blocking as broad as possible. In this case we can try to fix the breakage on your sites.

Maybe this will work https://github.com/easylist/easylist/pull/19367

**Comment by smalluban** [comment]
I am closing the issue, as the it relates to the filter maintained by the community list, but not to the extension codebase.
